conda-forge.yml

```
@@ -1,6 +1,12 @@
-github:
-  branch_name: main
-  tooling_branch_name: main
+azure:
+  free_disk_space: true
+#build_platform:  # force full emulation to resolve segfault from some libs
+#  linux_aarch64: linux_64
```

We can try the suggestions above with cross-compilation enabled (this change plus a re-render)

```suggestion
build_platform:
  linux_64: linux_64
  linux_aarch64: linux_64
```

---

Will do but in a new PR because holoscan team is OK for us to support `sbsa` starting with the latest GA version which is now `3.1.0.3`. I will refer to this PR from that, and close this PR later.